<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>event_information</class>
 <widget class="QMainWindow" name="event_information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>60</y>
      <width>191</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="12" column="1">
      <widget class="QPushButton" name="book_info_but">
       <property name="text">
        <string>书籍管理</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QPushButton" name="user_info_but">
       <property name="text">
        <string>用户管理</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="search_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="search_but">
         <property name="text">
          <string>搜索</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>书号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="book_id_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="book_id_search_but">
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>查看</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>账号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="user_id_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="user_id_search_but">
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>查看</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>编号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="id_edit"/>
       </item>
      </layout>
     </item>
     <item row="15" column="1">
      <widget class="QPushButton" name="exit_but">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QRadioButton" name="user_name_but">
         <property name="text">
          <string>用户名</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="book_name_but">
         <property name="text">
          <string>书名</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="book_id_but">
         <property name="text">
          <string>书号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="user_id_but">
         <property name="text">
          <string>帐号</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="insert_but">
       <property name="text">
        <string>借阅</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="log_but">
       <property name="text">
        <string>日志</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QPushButton" name="modify_but">
       <property name="text">
        <string>修改个人信息</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QPushButton" name="return_but">
       <property name="text">
        <string>归还</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="event_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>741</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <column>
     <property name="text">
      <string>编号</string>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>帐号</string>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>用户名</string>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>书号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>书名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>借阅时间</string>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>归还时间</string>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
